### ISSUE TITLE
IDOR remediation unrealistic?

### ISSUE THREAD

**Issue by sea-erkin** [righettod/poc-idor#2]
Isn't it inefficient to loop through all of the movies and calculate the hash each time the user wants to retrieve just an individual movie? I understand why this is being done and it can be a quick check with a small dataset, but doesn't it seem like recommending this approach to remedy IDOR is a little unrealistic for most people?

**Comment by righettod** [comment]
I agree,  the proposal was done for small dataset. For large or medium
dataset, it is better to assign a technical id in a field that is
precomputed at insert time and use it as id in the front end in order to
avoid over computation.
